contracts/ReversibleICO.sol

```diff
@@ -234,7 +234,7 @@ contract ReversibleICO is IERC777Recipient {
         // Assign other variables
         commitPhaseStartBlock = _commitPhaseStartBlock;
         commitPhaseBlockCount = _commitPhaseBlockCount;
-        commitPhaseEndBlock = _commitPhaseStartBlock + _commitPhaseBlockCount;
+        commitPhaseEndBlock = _commitPhaseStartBlock + _commitPhaseBlockCount - 1;
```

Comment:
This seems wrong, as the last block of the phase is now -1

Reply:
I don't think so, if the startBlock is 1 and the duration is 12 then the endBlock is 12 (1+12-1)

Reply:
if startBlock is 11 and duration is 10, end should be 21
end = start + count - 1 => is the correct one if duration includes start and end blocks.